`make lint` does not work locally

**Describe the bug**
`make lint` does not work when the user has a .venv folder.

**To reproduce**
* Create virtual environment in monitoring/.venv
* Run `make lint`
* Observe something similar to the below:

```
make lint
find . -name '*.sh' ! -path "./interfaces/*" | xargs docker run --rm -v "/usr/local/google/home/pelletierb/IdeaProjects/OpenSource/monitoring:/monitoring" -w /monitoring koalaman/shellcheck:v0.11.0

In ./.venv/lib/python3.13/site-packages/nodejs_wheel/lib/node_modules/npm/lib/utils/completion.sh line 21:
    if ! IFS=$'\n' COMPREPLY=($(COMP_CWORD="$cword" \
                              ^-- SC2207 (warning): Prefer mapfile or read -a to split command output (or quote to avoid splitting).


In ./.venv/lib/python3.13/site-packages/nodejs_wheel/lib/node_modules/npm/lib/utils/completion.sh line 39:
    compadd -- $(COMP_CWORD=$((CURRENT-1)) \
               ^-- SC2046 (warning): Quote this to prevent word splitting.


In ./.venv/lib/python3.13/site-packages/nodejs_wheel/lib/node_modules/npm/lib/utils/completion.sh line 50:
    read -Ac words
    ^--^ SC2162 (info): read without -r will mangle backslashes.


In ./.venv/lib/python3.13/site-packages/nodejs_wheel/lib/node_modules/npm/lib/utils/completion.sh line 51:
    read -cn cword
    ^--^ SC2162 (info): read without -r will mangle backslashes.


In ./.venv/lib/python3.13/site-packages/nodejs_wheel/lib/node_modules/npm/lib/utils/completion.sh line 52:
    let cword-=1
    ^----------^ SC2219 (style): Instead of 'let expr', prefer (( expr )) .


In ./.venv/lib/python3.13/site-packages/nodejs_wheel/lib/node_modules/npm/lib/utils/completion.sh line 53:
    read -l line
    ^--^ SC2162 (info): read without -r will mangle backslashes.


In ./.venv/lib/python3.13/site-packages/nodejs_wheel/lib/node_modules/npm/lib/utils/completion.sh line 54:
    read -ln point
    ^--^ SC2162 (info): read without -r will mangle backslashes.


In ./.venv/lib/python3.13/site-packages/nodejs_wheel/lib/node_modules/npm/lib/utils/completion.sh line 56:
    if ! IFS=$'\n' reply=($(COMP_CWORD="$cword" \
                   ^---^ SC2034 (warning): reply appears unused. Verify use (or export if used externally).
                          ^-- SC2207 (warning): Prefer mapfile or read -a to split command output (or quote to avoid splitting).

For more information:
  https://www.shellcheck.net/wiki/SC2034 -- reply appears unused. Verify use ...
  https://www.shellcheck.net/wiki/SC2046 -- Quote this to prevent word splitt...
  https://www.shellcheck.net/wiki/SC2207 -- Prefer mapfile or read -a to spli...
make: *** [Makefile:45: shell-lint] Error 123
```

**Difference from expected behavior**
`make lint` should work even if a local virtual environment exists

**Possible solution**
Ensure .venv (and any other ignored folders) are excluded from checks, and/or only mount non-ignored folders into container doing the linting.

--- a/Makefile
+++ b/Makefile
@@ -42,7 +42,7 @@ validate-uss-qualifier-docs:
 
 .PHONY: shell-lint
 shell-lint:
-	find . -name '*.sh' ! -path "./interfaces/*" | xargs docker run --rm -v "$(CURDIR):/monitoring" -w /monitoring koalaman/shellcheck:v0.11.0
+	find . -name '*.sh' ! -path "./interfaces/*" | git check-ignore --stdin --no-index -n -v --non-matching | grep '^::' | cut -f2 | xargs docker run --rm -v "$(CURDIR):/monitoring" -w /monitoring koalaman/shellcheck:v0.11.0
 
 .PHONY: unit-test
 unit-test:

